Source/LuaBridge/RefCountedPtr.h

```diff
@@ -159,13 +159,17 @@ class RefCountedPtr : private detail::RefCountedPtrBase
     */
     void reset()
     {
-        if (m_p != 0)
+        auto myCount_it = getRefCounts().find(m_p); // use find() to avoid looking up m_p more than once
```

Comment:
I still don't get the purpose/meaning of this comment. I would simply remove it.
Furthermore I would avoid auto because it's not used yet in this project and rename the variable to be more precise but this is my personal flavor/feeling.
`const RefCountsType::iterator itCounter = getRefCounts().find(m_p);`

Reply:
Lol, well, the codebase is committed to C++11 and `auto` is part of that. But being new here, I will defer to your preference. Personally, I've come to quite like `auto` and use it with increasing frequency.

Reply:
For me both variants are fine but I usually try to stick to the project convention if possible to keep source code similar.